ostd/src/orpc/oqueue/generic_test.rs

```diff
@@ -205,7 +226,11 @@ pub(crate) fn test_send_multi_receive_blocker<T: OQueue<TestMessage>>(
     // Receiver which receives all the messages
     let receiver1 = oqueue1.attach_receiver().unwrap();
     let receiver2 = oqueue2.attach_receiver().unwrap();
-    let receive_thread = thread::spawn({
+    let recv_queue = Arc::new(WaitQueue::new());
+    let recv_queue_cloned = recv_queue.clone();
+    let recv_completed = Arc::new(AtomicBool::new(false));
+    let recv_completed_cloned = recv_completed.clone();
+    let receive_thread = TaskOptions::new({
         move || {
             let mut receiver1_counter = 0;
```

Comment:
This pattern of `*_cloned` stuff is a real wart is Rust. But I think it's better to put the clones inside the call you are passing the closure too. I put a note about it here:

https://github.com/ldos-project/asterinas/blob/main/docs/src/to-contribute/style-guidelines/rust-guidelines.md